examples/tutorials/nb_python/ECCV_2020_Navigation.py

```diff
@@ -53,30 +53,28 @@
 import cv2
 import git
 import magnum as mn
-# %matplotlib inline
-import matplotlib.pyplot as plt
 import numpy as np
+# %matplotlib inline
+from matplotlib import pyplot as plt
 # function to display the topdown map
 from PIL import Image
 
-import habitat_sim.utils.common as utils
+from habitat_sim.utils import common as utils
 from habitat_sim.utils import viz_utils as vut
 
 try:
```

Comment:
This fancy import pattern is no longer needed btw. You can just do import habitat_sim now.